core/src/edu/asu/c3simulator/simulation/Employee.java

```diff
@@ -0,0 +1,259 @@
+				"Baker", "Tyke", "Mattingly", "Feng", "Lee", "Smith", "Houston",
+				"Franco", "White", "Andrade", "Manning", "Brady", "Boyle", "Terry" };
+		
+		int randomFirst = (int) (Math.random() * firstName.length);
+		int randomLast = (int) (Math.random() * lastName.length);
+		
+		String randomEmployeeName = firstName[randomFirst] + " " + lastName[randomLast];
+		
+		return randomEmployeeName;
+	}
+	
+	public void setRandomEmployeeName()
+	{
+		this.employeeName = getRandomEmployeeName();
+	}
+	
+	public static String getEmployeePosition()
+	{
+		String[] positions = { "Manager", "Product Design", "Sales Associate",
+				"Marketing" };
+		int randomPositions = (int) (Math.random() * positions.length);
+		employeePosition = positions[randomPositions];
+		return employeePosition;
+	}
+	
+	public void setEmployeePosition(String employeePosition)
+	{
+		this.employeePosition = employeePosition;
+	}
+	
+	public int getHourlyWage()
+	{
+		return employeePay;
+	}
+	
+	public void setHourlyWage(int employeePay)
+	{
+		this.employeePay = employeePay;
+	}
+	
+	/**
+	 * Preferred Hourly Wage is determined by the selected employee's position and returns
+	 * an integer between the given pay range
+	 */
+	public static int getPreferredHourlyWage()
+	{
+		if (employeePosition == "Manager")
+		{
+			Random r = new Random();
+			int lowPay = 17;
+			int highPay = 24;
+			int i = r.nextInt(highPay - lowPay);
+			Employee.employeePreferredHourlyWage = i + lowPay;
+		}
+		
+		else if (employeePosition == "Product Design")
+		{
+			Random r = new Random();
+			int lowPay = 14;
+			int highPay = 19;
+			int i = r.nextInt(highPay - lowPay);
+			employeePreferredHourlyWage = i + lowPay;
+		}
+		
+		else if (employeePosition == "Sales Associate")
+		{
+			Random r = new Random();
+			int lowPay = 11;
+			int highPay = 16;
+			int i = r.nextInt(highPay - lowPay);
+			employeePreferredHourlyWage = i + lowPay;
+		}
+		
+		else if (employeePosition == "Marketing")
+		{
+			Random r = new Random();
+			int lowPay = 8;
+			int highPay = 13;
+			int i = r.nextInt(highPay - lowPay);
+			employeePreferredHourlyWage = i + lowPay;
+		}
+		
+		return employeePreferredHourlyWage;
+	}
+	
+	public void setPreferredHourlyWage(int emoloyeePreferredHourlyWage)
+	{
+		this.employeePreferredHourlyWage = emoloyeePreferredHourlyWage;
+	}
+	
+	/**
+	 * Employee Morale is determined by the difference between the employees pay
+	 * preference and their actual pay. If their morale reaches zero, the employee quits
+	 * and is removed from the active list of hired employees.
+	 */
+	public int getMorale()
+	{
+		int employeeMorale = 10;
+		final int MAX_DIFFERENCE = 4; // Largest difference an employee will take before
+										// quitting
+		
+		int payDifference = employeePreferredHourlyWage - employeePay;
+		
+		if (employeePay <= employeePreferredHourlyWage)
+		{
+			employeeMorale -= (payDifference * 2);
+		}
+		
+		if (employeePay >= employeePreferredHourlyWage)
+		{
+			employeeMorale = 10;
```

Comment:
Magic number. I think this is the max moral. If so, add a constant in Employee (Employee.MAXIMUM_MORAL)